piksi_tools/console/settings_view.py

```diff
@@ -223,34 +199,39 @@ def revert_to_prior_value(self, name, old, new, error_value=SBP_WRITE_STATUS.TIM
                  "   Error Value: {2}")
         invalid_setting_prompt.text = invalid_setting_prompt.text.format(self.name, new, error_value)
         invalid_setting_prompt.run()
+        self.reverting = False
```

Comment:
Are we sure we want to keep reverting true for this long time?  I feel like there were corner cases on the pop-up that caused odd behavior and that is why I turned reverting back off as soon as the old value was written to setting.  If you've adequately tested, I'm okay with the change, but I would prefer to unset .reverting as soon as we have done the revert to prior value.

Reply:
I have tested changing the unsupported `qzss_acquisition_enabled` setting and have not encountered any anomalies this far. The previous version did reverting in a new thread which can complicate things. In here all happens in the same work thread if the write fails.